.php_cs

```diff
@@ -5,7 +5,6 @@ $finder = Symfony\CS\Finder\DefaultFinder::create()
 ;
 
 return Symfony\CS\Config\Config::create()
-    ->level(Symfony\CS\FixerInterface::SYMFONY_LEVEL)
```

Comment:
Yeah ok. This is why I want to completely remove this dependencies and use StyleCI.

See here: https://github.com/sonata-project/SonataAdminBundle/pull/3273